doc/user/quick-sql-grammar.md

```diff
@@ -260,12 +270,12 @@ name.
 By default all DATE columns created using the Oracle DATE datatype. Use this
 setting to override this default.
 
-### db
+<!--### db
 
 **Possible Values**: `11g`, `12c`, `19c`, `21c`
 **Default Value**: `21c`
 
-Specifies the database version the syntax should be compatible with.
+Specifies the database version the syntax should be compatible with.-->
```

Comment:
Please bring `db` setting back, and get rid of `boolean` setting as discussed internally

Reply:
changed default to 19c